packages/amplify-e2e-core/src/configure/index.ts

```diff
@@ -47,6 +47,44 @@ export function amplifyConfigure(settings: AmplifyConfiguration): Promise<void>
     throw new Error(`mandatory params ${missingParam.join(' ')} are missing`);
   }
 
+  return new Promise((resolve, reject) => {
+    const chain = spawn(getCLIPath(), ['configure'], { stripColors: true })
+      .wait('Sign in to your AWS administrator account:')
+      .wait('Press Enter to continue')
+      .sendCarriageReturn()
+      .wait('Specify the AWS Region');
+
+    singleSelect(chain, s.region, amplifyRegions);
+
+    chain
+      .wait('Press Enter to continue')
+      .sendCarriageReturn()
+      .wait('accessKeyId')
+      .pauseRecording()
+      .sendLine(s.accessKeyId)
+      .wait('secretAccessKey')
+      .sendLine(s.secretAccessKey)
+      .resumeRecording()
+      .wait('Profile Name:')
+      .sendLine(s.profileName)
+      .wait('Successfully set up the new user.')
+      .run((err: Error) => {
+        if (!err) {
+          resolve();
+        } else {
+          reject(err);
+        }
+      });
+  });
+}
+
+export const amplifyConfigureBeforeV10_7 = (settings: AmplifyConfiguration): Promise<void> => {
```

Comment:
Now that 10.7 is released, should this be 10.8?

Or instead, rename this to something like `amplifyConfigureV10_7AndBelow`?